src/mats-method-workspace/mats-method-checks.service.ts

```diff
@@ -70,7 +70,7 @@ export class MatsMethodChecksService {
 
       if (beginDate && endDate.isBefore(beginDate)) {
         error = this.getMessage('MATSMTH-3-A');
-      } else if (endDate.isAfter(moment(getMaximumFutureDate()))) {
+      } else if (endDate.isAfter(moment(getMaximumFutureDate))) {
```

Comment:
This _should_ call the function:
```suggestion
      } else if (endDate.isAfter(moment(getMaximumFutureDate()))) {
```